proxy/invoke_test.go

```diff
@@ -15,22 +15,42 @@ import (
 )
 
 func Test_InvokeFunction(t *testing.T) {
-	s := test.MockHttpServerStatus(t, http.StatusOK)
+	s := test.MockHttpServerStatus(t, http.StatusOK, http.StatusOK)
 	defer s.Close()
 
-	bytesIn := []byte("test data")
-	_, err := InvokeFunction(
-		s.URL,
-		"function",
-		&bytesIn,
-		"text/plain",
-		[]string{},
-		[]string{},
-	)
-
-	if err != nil {
-		t.Fatalf("Error returned: %s", err)
-	}
+	t.Run("Sync", func(t *testing.T) {
```

Comment:
Generally I think we're using test-scenarios over `t.Run` for consistency.. Or is `t.Run` widely used in the tests and I missed it?

Reply:
Have reverted to individual tests, as an fyi tho I strongly prefer table driven/multi part tests in order to both minimise repetition, and unnecessary test case setup.

Reply:
t.Run is how table driven tests work btw